### PR TITLE
add compiled definition support

### PR DESCRIPTION
**Pull request by djkelleher**
Passing precompiled view definitions to `CreateView` is useful when working with dialect-specific queries. This branch adds support for this via a simple type check.

### CODE REVIEW

**Comment by jklukas** [comment]
It looks like you've run this through a formatter, which makes it difficult to distinguish the functional changes from stylistic changes. Can you recreate this with just the functional changes you're making?

**Comment by djkelleher** [comment]
@jklukas done

File: tests/test_views.py [hunk 1]
```diff
@@ -152,4 +164,4 @@ def test_drop_with_on_parameter_is_not_none():
     """
     view = Table('myview', sa.MetaData())
     with pytest.raises(TypeError):
-        DropView(view, on=True)  # on is not None
+        DropView(view, on=True)  # on is not None
```
Comment (jklukas) [inline tests/test_views.py]:
```suggestion
        DropView(view, on=True)  # on is not None

```

**Review by jklukas — APPROVED** [review]
Thanks for reverting the formatting, and thanks for the contribution. This change looks like a nice improvement and low risk of affecting existing use cases.

**Comment by jklukas** [comment]
0.3.1 is now released with this change.

**Comment by djkelleher** [comment]
Thanks for the review/merge/release! @jklukas

**Pull request merged**